R/MltplxObject.R

```diff
@@ -258,7 +258,8 @@ update_object_intensity = function(mltplx_object, ps, bw){
 #' name of the distance metric you use
 #' @export
 update_object_dist = function(mltplx_object, dist_metric,
-                              .dist_metric_name = NULL){
+                              .dist_metric_name = NULL,
+                              symmetric = TRUE){
```

Comment:
`symmetric` should be passed as a parameter to MltplxObject and MltplxExperiment too, right?

Reply:
Yup- fixed